source/helpers/test/CaseDataConverter.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { replaceTagPart } from '../CaseDataConverter';
+
+describe('replaceTagPart', () => {
+  
+  it('should replace part in tag on single occurence', () => {
```

Comment:
Many people write **it('should'**...). ..which I find strange. When using the word `should` you are basically saying that the test could have more than one outcome, but it shouldn't because that would be a failure.

So in this particular test the meaning could be something like this:
`it('replaces the tag part on a single occurence'...`

Reply:
From previous testing i Jest i have learned to view the test as a check for a function. And this check cannot guarantee that the outcome will be true every time. Therefore you can declare this in the test name with the prefix should or expect. This also ease the naming of tests when writing them since you always start the sentence with a keyword.

But I'm not the one to hold on to previous knowledge, so if we se this prefixing as something unnecessary or confusing then i yeild myself :D